324: Automatische Übernahme Baujahr aus NHK2010 in Grunddaten

joergreichert opened the pull request:
erweitern 

Task-Url: http://polymap.org/kaps/ticket/324

Setze Baujahr und Baujahr bereinigt automatisch auf die Werte aus
eventuell vorhandener NHK 2010 Bewertungen

joergreichert committed:
330: Suchfunktion um Nutzungsgröße bei erweiterten Vertragsdaten Agrar
erweitern 

Task-Url: http://polymap.org/kaps/ticket/330

Setze Baujahr und Baujahr bereinigt automatisch auf die Werte aus
eventuell vorhandener NHK 2010 Bewertungen

joergreichert committed:
330: Suchfunktion um Nutzungsgröße bei erweiterten Vertragsdaten Agrar
erweitern 

Task-Url: http://polymap.org/kaps/ticket/330

Summe Wohnflächen aus Ertragswertverfahren beziehen

stundzig commented on plugins/org.polymap.kaps/src/org/polymap/kaps/ui/form/VertragsdatenBaulandGrunddatenFormEditorPage.java:
```diff
@@ -167,6 +159,24 @@ public void run() {
+        	}
+        }
+        if(bereinigtesBaujahr != null) {
+        	vb.baujahrBereinigt().set(bereinigtesBaujahr.intValue());
```
Hier ebenfalls, Berechnung auslagern in separate Methode und nicht am Entity speichern sondern mit _site.setFieldValue()_... setzen.

joergreichert committed:
324: Automatische Übernahme Baujahr aus NHK2010 in Grunddaten 

Task-Url: http://polymap.org/kaps/ticket/324

Setze Baujahr-Felder erst nach Erzeugen des Forms und auch nur in den
Feldern, nicht am Modell.
